src/components/Nav/SearchBar.jsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function SearchBar(props) {
+  // display: ${props.display ? 'flex' : 'none'};
+  const DropDown = styled.div`
+    position: absolute;
+    top: 91px;
+    width: 100%;
+    border: 1px solid black;
+    display: ${props.display ? 'flex' : 'none'};
+    flex-direction: row;
+    justify-content: center;
+    padding: 50px 400px;
+    background-color: #372667;
+  `;
+
+  const SubWrapper = styled.div`
+    width: 1100px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: start;
+  `;
+
+  const SortingWrapper = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-bottom: 40px;
+  `;
+
+  const SortingOrder = styled.span`
+    flex-shrink: 0;
+    color: white;
+    margin-right: 30px;
+    padding-bottom: 5px;
+    border-bottom: 2px solid white;
+    font-size: 15px;
+  `;
+
+  const HeaderSearchWrapper = styled.div`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  `;
+
+  const ImageWrapper = styled.div`
+    width: 150px;
+    height: 214px;
+    margin-right: 50px;
+  `;
+  const MovieImage = styled.img`
+    max-width: 100%;
+  `;
+
+  const MovieRank = styled.span`
+    font-style: italic;
+    font-size: 25px;
+    margin-right: 10px;
+    font-family: Roboto;
+  `;
+
+  const MovieTitle = styled.a`
+    flex-shrink: 0;
+    font-size: 15px;
+  `;
+
+  const Movie = styled.li`
+    color: white;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  `;
+
+  const MovieList = styled.ul`
+    padding-right: 80px;
+  `;
+
+  const SearchWrapper = styled.div`
+    position: relative;
+    width: 450px;
+  `;
+
+  const SearchBar = styled.input`
+    width: 450px;
+    padding: 20px 10px;
+    font-size: 18px;
+    border: 0;
+    color: white;
+    border-bottom: 2px solid white;
+    background-color: #372667;
+    ::placeholder {
+      color: white;
+      opacity: 1; /* Firefox */
+    }
+  `;
+
+  const SearchIcon = styled.span`
+    position: absolute;
+    left: 92%;
+    top: 25%;
+    font-size: 22px;
+    color: white;
+  `;
+
+  return (
+    <DropDown>
+      <SubWrapper>
+        <SortingWrapper>
+          <SortingOrder>예매율 순</SortingOrder>
+          <SortingOrder>메가박스 관객순</SortingOrder>
+        </SortingWrapper>
+        <HeaderSearchWrapper>
+          <ImageWrapper>
+            <MovieImage
+              src={
+                'https://img.megabox.co.kr/SharedImg/2021/09/07/3UeqSChjYVBdtycHCNm5fkNhlZeDAWps_316.jpg'
+              }
```

Comment:
```suggestion
              src='https://img.megabox.co.kr/SharedImg/2021/09/07/3UeqSChjYVBdtycHCNm5fkNhlZeDAWps_316.jpg'
```
중괄호가 불필요하게 사용되었습니다 유사한 부분도 같이 고쳐주세요

Reply:
중괄호가 불필요하게 사용된 부분들은 삭제하겠습니다.

Reply:
@JackAwesomeKim 동준님 이거 반영 안된거 같아요